ingredient-service/src/test/java/eu/innowise/ingredientservice/testutil/builders/IngredientTestBuilder.java

```diff
@@ -0,0 +1,53 @@
+package eu.innowise.ingredientservice.testutil.builders;
```

Comment:
what??? why we need this? Are you testing lombok?) maybe in next PR you will testing Spring?